src/js/components/Grommet/StyledGrommet.js

```diff
@@ -7,6 +7,7 @@ const fullStyle = (full) => {
   if (full === 'min')
     return css`
```

Comment:
Can we add a comment above the return, so as not to be in the generated code, that explains why we have the `height: 1px` here? I suspect this will help us when we forget down the road.